### DISCUSSION TITLE
Add settting to change "auto rewind" amount when interrupted by other apps

### DISCUSSION THREAD

**Discussion by k9withabone** [PaulWoitaschek/Voice#1877] [Ideas]
App-Version: 4.3.2
Phone Name: OnePlus 6
Android Version: 9 (OxygenOS 9.0.7)

As noted in #618, a fixed 1s "auto rewind" was added for when Voice is interrupted by other apps. I was thinking of adding an option to be able to change this value and/or sync it with the normal auto rewind setting. I know in #618 it was said that this was not done by default because if multiple notifications are coming in then the auto rewind would stack to a significant amount. I think I might be able to figure out a way so that the auto rewind is only triggered once when multiple notifications are coming in. However, in the meantime, I would like to add the option to change the interrupted auto rewind amount, perhaps in advanced settings section to not confuse a normal user.

**Comment by jtagcat** [comment]
IIRC there's also a way to block most interrupts.
